packages/client/hmi-client/src/services/model.ts

```diff
@@ -1,6 +1,11 @@
 import API from '@/api/api';
 import { Model } from '@/types/Model';
 
+export async function createModel(model): Promise<Model | null> {
+	const response = await API.post(`/models`, model);
+	return response?.data ?? null;
+}
```

Comment:
"Model type as defined int he UI is not the same as the Model payload sent into the API" exactly the reason to not type the input as a Model, because Model requires a bunch of mandatory fields (for example id) that don't make sense to send for model creation.